packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-server/table-render-10k.benchmark.js

```diff
@@ -7,10 +7,10 @@
 
 import { renderComponent } from '@lwc/engine-server';
 
-import Table from '@lwc/perf-benchmarks-components/dist/server/benchmark/cardComponent/cardComponent.js';
+import Table from '@lwc/perf-benchmarks-components/dist/server/benchmark/table/table.js';
 import Store from '@lwc/perf-benchmarks-components/dist/server/benchmark/store/store.js';
 
-benchmark(`server/table/render/10k`, () => {
+benchmark(`server/table-v2/render/10k`, () => {
```

Comment:
This was a bug in the previous benchmark. I don't think we meant to actually render the Card component here.

To avoid our Best data looking weird (i.e. big change after this commit), I'm just renaming the benchmark entirely to `v2`.